tests/installation/releasenotes.pm

```diff
@@ -64,6 +64,13 @@ sub run {
         push @no_relnotes, qw(geo);
     }
     if (@addons) {
+        # Workaround release notes issues in upgrade sle12+addons to sle15
+        # The workaround must be removed after bugs fixed
+        if (get_var('UPGRADE') && sle_version_at_least('15')) {
```

Comment:
I have a small concern here, as sle_version_at_least returns true for all non-sle distributions. I've checked that we don't have problems with openSUSE. To play it safe I would add is_sle check, but if you confirm that it's not executed for other distris, we can merge it.

Reply:
Test with a tw scenario to be sure

Reply:
Don't worry, seems opensuse doesn't check release notes so far. I checked products/opensuse/main.pm, and also verified with local test runs, such as:
   * update_Leap_42.3_gnome on Leap-15.0: http://openqa-apac1.suse.de/tests/125
   * update_Leap_42.3_gnome on Tumbleweed: http://openqa-apac1.suse.de/tests/126